src/main/java/com/example/videotecha/dto/ErrorDto.java

```diff
@@ -0,0 +1,43 @@
+package com.example.videotecha.dto;
+
+import org.springframework.http.HttpStatus;
+
+import java.sql.Timestamp;
```

Comment:
This class is used within the JDBC API and probably shouldn't be used in your DTOs.